### ISSUE TITLE
Username capitalization when commenting

### ISSUE THREAD

**Issue by jdaudier** [loicginoux/joanne_project#120]
Let me know if this is easy enough to put under sprint 6 or else I can move it to sprint 7.

http://screencast.com/t/bwFADI7uKjCe

**Comment by loicginoux** [comment]
ok for sprint 6

**Comment by jdaudier** [comment]
Related update for email notifications: http://screencast.com/t/D85O7miuzmip

**Comment by loicginoux** [comment]
I just put the label "in testing". For the issues with this label, just close them if it's working for you. I will test them first and check it's working for me before putting them "in testing"

**Comment by jdaudier** [comment]
Ok the comment on site is working.  Is the email updated too?

**Comment by loicginoux** [comment]
nope

**Comment by jdaudier** [comment]
ok - once email is updated, I can close this

**Comment by loicginoux** [comment]
If you mean, the bug when you receive an email when you comment you own photo and you should not. It should go to another issue (If not yet created).

**Comment by jdaudier** [comment]
I meant this: http://screencast.com/t/D85O7miuzmip

**Comment by jdaudier** [comment]
Updated: http://screencast.com/t/hypTsaAxJ

**Comment by jdaudier** [comment]
Can you confirm this is done since I can't see it since I don't have any new followers: http://screencast.com/t/hypTsaAxJ

**Comment by jdaudier** [comment]
Can we update this too? http://screencast.com/t/oFaKy1LM

**Comment by jdaudier** [comment]
One more I noticed: http://screencast.com/t/5AHP3juD

**Comment by jdaudier** [comment]
I think we can close this once this is done: http://screencast.com/t/5AHP3juD
